src/components/transport_manager/src/usb/libusb/usb_handler.cc

```diff
@@ -501,5 +505,15 @@ void UsbHandler::UsbHandlerDelegate::threadMain() {
   handler_->Thread();
 }
 
+void UsbHandler::UsbHandlerDelegate::exitThreadMain() {
+  handler_->shutdown_requested_ = true;
```

Comment:
@anosach-luxoft This thread delegate is initialized as `new UsbHandlerDelegate(this)` in the constructor of `UsbHandler` which supplies valid pointer.